dangerzone/gui/main_window.py

```diff
@@ -112,7 +117,7 @@ def closeEvent(self, e: QtGui.QCloseEvent) -> None:
             else:
                 e.accept()
 
-        self.dangerzone.app.quit()
+        self.dangerzone.app.exit(0)
```

Comment:
I'd personally change the return code to 2, since it's not a successful run. Other than that, LGTM.

Reply:
Sounds good. I have now changed it.